### ISSUE TITLE
[NPC][Wotlk] Kvaldir Mist Lord and Kvaldir Mistweaver

### ISSUE THREAD

**Issue by Teridan-dal-wow** [dalaranwow/dalaran-wow#4551]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: 
Npc http://wotlk.openwow.com/npc=25496 is missing 2 mini events, some paths, spawns and emotes
Npc http://wotlk.openwow.com/npc=25479 is missing some spawns and has wrong state

**Expected behaviour**: Video from retail legion from 10.3.2017 for the 2-nd mini event only
https://www.youtube.com/watch?v=W0pE594frW4&feature=youtu.be

Note: The second mini event is done only by few creatures ( `in (110298, 110300, 110294, 5005569, 5005568);` )  with npc 110298 is doing the mini event every  ~ 20 secs , while the rest every ~ 60 secs.

What is done so far:
-added the missing spawns
-added the missing paths and emotes
-added 1 mini event
-set few creatures to reach defensive after checking on retail
-added 1 combat spell
-added the text for the 2-nd mini event

To do:
-npc http://wotlk.openwow.com/npc=25504 has a script related to quest http://wotlk.openwow.com/quest=11664 and after i added flags =768 to her (checked on retail) her script needs to be edited to remove these flags on quest start. Also she should have reach = defensive while she is waiting for someone to start her quest (currently its aggresive). 

-add the 2-nd mini event

-fix an issue with npc http://wotlk.openwow.com/npc=25479 .They had inhabittype= 7 which caused issues when they enter the water and go in deep place they start to swim. That should not happen, on retail they are flying, so set their inhabittype = 4 , which fixed the issue and it looks exacly like on retail, however it caused a new issue. They i pull them and i am flying they start to fly also, thats not correct, they should behaive as ground creature and reset it they can't reach me.
![image](https://cloud.githubusercontent.com/assets/6797611/23794760/73e5544c-059a-11e7-9e9c-c6dd342ea9af.png)

How it looks so far:
![image](https://cloud.githubusercontent.com/assets/6797611/23794793/a45d242e-059a-11e7-8b60-47bf5194802c.png)
![image](https://cloud.githubusercontent.com/assets/6797611/23794800/b0fdec04-059a-11e7-9c7b-07db18fa2a90.png)
![image](https://cloud.githubusercontent.com/assets/6797611/23794809/c6e38330-059a-11e7-9b3a-bf5a41ab614c.png)
![image](https://cloud.githubusercontent.com/assets/6797611/23794823/e2173c82-059a-11e7-8c6d-9483a5e22c2d.png)

**Comment by Rushor** [comment]
i already scripted the second minievent in tc, but i think it got lost over the years xd will fix that

**Comment by Rushor** [comment]
DB/Creature: Kvaldir Mist Lord and Kvaldir Mistweaver -- Add missing spawns, paths, emotes and one minievent

* updates: https://github.com/dalaranwow/dalaran-wow/issues/4551
* commit: c75e6a74075ae3193e9e78ac5cfad62bcc39f8d8

**Comment by Teridan-dal-wow** [comment]
> i already scripted the second minievent in tc, but i think it got lost over the years xd will fix that

There was only 1 script regarding this area and it was for the guy around the fire with waypoint script, but that event is not correct to be done that way, so i deleted it :)

**Comment by Rushor** [comment]
also need to add trigger for this in order to match the orientation

**Comment by Rushor** [comment]
DB/Creature: Kvaldir Mist Lord - Add Movement Script + Add inhabittypescript for Kvaldir Mistweaver + Update flags and reactstate of Mootoo the Younger on reset and queststart

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4551
* commit: ef3b25f88f940d1bfd48b03fd784150f84efd30e

![GitHub Logo](http://i.imgur.com/M2eIahg.jpg)
![GitHub Logo](http://i.imgur.com/cK09bB4.jpg)
![GitHub Logo](http://i.imgur.com/dxscA29.jpg)
![GitHub Logo](http://i.imgur.com/GAUSUZe.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
